src/goodbase/goodbase_cli.py

```diff
@@ -439,8 +439,8 @@ def main(
         branch_name=branch,
         output_format=output_format,
     )
-    if display_variant_name:
-        display_variant_name_checks = display_variant_name
+
+    display_variant_name_checks = display_variant_name
```

Comment:
This probably won't fix the bug user reported, because he didn't pass `display_variant_name` explicitly, so the condition `if display_variant_name is not None` [here](https://github.com/evergreen-ci/git-co-evg-base/blob/49584ccaac7ba9c51ce354b5c3575bea9ecb7949/src/goodbase/goodbase_cli.py#L470) should have been evaluated `False`.  Maybe it's an empty list or something, not `None`. We should probably change it to `if display_variant_name`.